lib/shopify_api/resources/assigned_fulfillment_order.rb

```diff
@@ -0,0 +1,12 @@
+module ShopifyAPI
+  class AssignedFulfillmentOrder < Base
+    CANCELLATION_REQUESTED = 'cancellation_requested'
+    FULFILLMENT_REQUESTED = 'fulfillment_requested'
+    FULFILLMENT_ACCEPTED = 'fulfillment_accepted'
```

Comment:
What are these constants for?

Reply:
They are the possible `assignment_status` values that can be used in `options` for selecting returned FulfillmentOrders.

Reply:
Will be removing these constants and client-side checks. This will avoid duplication of logic to have to maintain sync.